### PR TITLE
Server logs, statuses and minor fixes

### PR DESCRIPTION
**Pull request by jkp85**
This PR changes server status behavior. Now status is fetched directly from docker daemon.
It also adds a new field to server json `logs_url` that is a websocket endpoint for logs stream.

### CODE REVIEW

**Comment by johngriebel** [comment]
@jkp85 looks fine to me.

File: base/management/commands/site_host.py [hunk 1]
```diff
@@ -8,7 +8,7 @@ class Command(BaseCommand):
     def handle(self, *args, **options):
         site = Site.objects.get(pk=settings.SITE_ID)
         host = os.environ.get('TBS_HOST')
-        if host:
+        if host and site.domain == 'example.com':
```
Comment (jgwerner) [inline base/management/commands/site_host.py]: Question: how come we have both site.name and site.domain both set to TBS_HOST?

File: docker-compose.yml [hunk 2]
```diff
@@ -38,15 +38,13 @@ services:
   server:
     image: 3blades/openresty:latest
     command: -p '' -c nginx.conf -g 'daemon off;'
+    env_file: env
     ports:
       - "5000:80"
     depends_on:
       - api
     volumes_from:
       - api
```
Comment (jgwerner) [inline docker-compose.yml]: So we can deprecate logspout repo from our stack?

Comment (jkp85) [inline docker-compose.yml]: Yes we can.

File: base/management/commands/site_host.py (hunk 1 above)
Comment (jkp85) [inline base/management/commands/site_host.py]: This is simply convenience. We can set site.name to whatever we want. Site domain is what makes things work.

File: servers/tests/test_views.py [hunk 3]
```diff
@@ -92,9 +93,10 @@ def test_server_delete(self):
         self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
         self.assertIsNone(Server.objects.filter(pk=server.pk).first())
 
-    def test_server_internal_running(self):
+    @patch('servers.spawners.DockerSpawner.status')
+    def test_server_internal_running(self, server_status):
+        server_status.return_value = Server.RUNNING
```
Comment (jgwerner) [inline servers/tests/test_views.py]: Should we add additional tests for other states?

Comment (jkp85) [inline servers/tests/test_views.py]: There is one, but not all statuses are tested. Pushed a fix for that.

File: docker-compose.yml (hunk 2 above)
Comment (jgwerner) [inline docker-compose.yml]: Ok once the PR is merged, we can confirm to deprecate 3blades/logspout. However, would need to confirm syslog forwards to internal or external services.

**Comment by jgwerner** [comment]
@jkp85 @johngriebel I connected this PR to #91. Will wait to merge until we make sure Issue has the necessary info.

**Pull request merged**